src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
             result.ShouldBe(0);
         }
 
-        [Fact]
-        public void StrykerCLI_WithNoFilesToExcludeSet_ShouldPassDefaultValueToStryker()
+        [Theory]
+        [InlineData("--help")]
+        [InlineData("-h")]
+        [InlineData("-?")]
+        public void ShouldNotStartStryker_WithHelpArgument(string argName)
```

Comment:
```suggestion
        public void ExecutedWithHelpArgument_ShouldNotStartStryker(string argName)
```
For tests naming consistency? (below as well).